tests/emm/services/test_git_service.py

```diff
@@ -227,4 +227,4 @@ def test_relative_directory_should_return_full_path(self, local_mock, git_servic
 
     def test_absolute_directory_should_return_directory(self, git_service):
         directory = Path("/a/absolute/path")
-        assert git_service._determine_directory(directory) == directory
+        assert git_service._determine_directory(directory) == Path.cwd() / directory
```

Comment:
This change is incorrect. The directory given here is an absolute path. It should not be prefixed by the CWD.